src/pages/HttpBin/index.js

```diff
@@ -34,33 +34,30 @@ const useStyles = makeStyles((theme) => ({
 const HttpBin = () => {
   const classes = useStyles();
   const [methodType, setCurrency] = useState('GET');
+  const [urlFormValue, setURLlFormValue] = useState('/get');
   const [queryParameters, setQueryParameters] = useState(
-    exampleJson.requestConfig,
+    JSON.stringify(exampleJson.queryParameters),
   );
+
   const [bodyParameters, setBodyParameters] = useState(
     JSON.stringify(exampleJson.data),
   );
   const [response, setResponse] = useState({});
   const [responseIsLoad, setResponseIsLoad] = useState(true);
 
-  const createRequest = (queryParameters) => {
+  const createRequest = () => {
     return new Promise((resolve, reject) => {
-      try {
-        const baseURL = 'https://httpbin.org/';
-        const queryObject = new URLSearchParams(queryParameters);
-        const pathObject = new URL(baseURL + queryParameters);
-
-        const url =
-          pathObject.pathname.length - 1 === 0
-            ? methodType.toLowerCase()
-            : pathObject.pathname;
+      const baseURL = 'https://httpbin.org/';
+      const url =
+        urlFormValue.length === 0 ? methodType.toLowerCase() : urlFormValue;
 
+      try {
         const axiosConfig = {
           method: methodType,
           baseURL: baseURL,
           data: bodyParameters,
           url: url,
-          params: queryObject,
+          params: JSON.parse(queryParameters),
         };
```

Comment:
do you use this variable in another place? If you don't, please try to avoid using useless variables.

Reply:
what variable did you write about?

Reply:
I'm asking about `axiosConfig` variable

Reply:
OK I understood. Update in the next commit